apis/nucleus/src/components/listbox/components/useTempKeyboard.js

```diff
@@ -0,0 +1,40 @@
+import { useState } from 'react';
+
+// Emulate the keyboard hook, until we support it in the Listbox.
+export default function useTempKeyboard({ containerRef, enabled }) {
+  const [keyboardActive, setKeyboardActive] = useState(false);
+
+  const innerTabStops = !enabled || keyboardActive;
+
+  const keyboard = {
+    enabled,
+    active: keyboardActive,
+    innerTabStops, // does keyboard permit inner tab stops
+    outerTabStops: !innerTabStops, // does keyboard permit outer tab stops
+    blur: (resetFocus) => {
+      setKeyboardActive(false);
+      const vizCell = containerRef.current?.closest('.njs-cell') || containerRef.current.closest('.qv-gridcell');
+      if (resetFocus && vizCell) {
+        // Move focus to the viz's cell.
+        vizCell.setAttribute('tabIndex', 0);
+        vizCell.focus();
+      }
```

Comment:
After more thinking. I this will be OK for filter pane.
But not in a listbox object. (or a filterpane with only one listbox)
then you don't want to have focus on the `.listbox-container` just go direct to the cell level

Reply:
Yeah, that's true. So, we would have to check if we have any sibling listboxes. If no siblings, then focus directly to the cell level. However, that logic would have to be handled by the listbox and not by the keyboard.

Reply:
I have implemented this for the Esc (blur) case:
`row -> [esc] -> cell`

But when returning back from cell to listbox it will still go through the listbox:
`cell -> [enter/space] -> listbox-container -> row`

Not sure yet where we host this focus logic in nebula but will leave for now.